packages/iTwinUI-react/.gitignore

```diff
@@ -0,0 +1,41 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+node_modules/
+node_modules/*
+
+# testing
+coverage/
+coverage/*
+
+# production
+build/
+build/*
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# Error/Debug logs
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+debug.log*
+
+# Compiled Components
+lib/
```

Comment:
so then we dont have lib anymore ... and same question about multiple gitignore files